src/kicanvas/elements/common/project-panel.ts

```diff
@@ -65,6 +65,7 @@ export class KCProjectPanelElement extends KCUIElement {
     ];
 
     #menu: KCUIMenuElement;
+    #selected_name: string | null = null;
```

Comment:
I'm not totally sure what the purpose of this attribute is. I tried removing it, and it seems to work just fine. I'd really appreciate it if you could explain it to me.

Reply:
Did you see my response to this?  I wrote one, but perhaps I forgot to save it.

If you click on a schematic name in the list on the right, it is visually highlighted.  That should also happen if we select a schematic with `sheet=`.  This code is necessary to highlight the sheet name if we use `sheet=`.

Reply:
I think there are indeed some imperfections here. Maybe something about `initialContentCallback` and it loaded but no event handler was registered. I would really recommend that we address the root cause of this issue. Well, anyway, it seems like this should be handled in another PR. After this issue is fixed, changes to file `project-panel.ts` will no longer be necessary. So we just accept its imperfections.